client/guidestours/steps.js

```diff
@@ -38,10 +38,10 @@ class GuidesFirstStep extends Component {
 
 		const { text, onNext, onQuit } = this.props;
 		return (
-			<Card className="guidestours__step" style={ stepCoords } >
+			<Card className="guidestours__step guidestours__step-first" style={ stepCoords } >
```

Comment:
The buttons also look different (not borderless, different wording). IMHO a case could be made for either route, but in any case we'd try to predict the future. I'd propose this is a change we should do when it's more obvious what the route to take is.